### PR TITLE
Feature/support require override

### LINKED ISSUE #581: [feature request] add support to require-override

### PR DESCRIPTION
**Pull request by boussaffawalid**
Feature: added support for require override 

closes #581

### CODE REVIEW

**Comment by boussaffawalid** [comment]
@uilianries is this project still maintained? the last commit I see here was 5 months ago. Is it possible to get some review for this PR

File: cpt/runner.py [hunk 1]
```diff
@@ -111,6 +114,8 @@ def run(self):
                 with tools.environment_append({"_CONAN_CREATE_COMMAND_": "1"}):
                     params = {"name": name, "version": version, "user": user,
                               "channel": channel, "build_modes": self._build_policy,
+                              "require_overrides": self._require_overrides,
+                              "re"
```
Comment (uilianries) [inline cpt/runner.py]:
```suggestion
```

**Review by uilianries — COMMENTED** [review]
Sorry for the delay. Please, only remove the leftover and we can proceed.

**Comment by boussaffawalid** [comment]
@uilianries I've rebased the PR and updated readme file.

**Comment by uilianries** [comment]
Almost all tests failed. Need to check if is something related to your PR or some external update which broke the project.

**Comment by boussaffawalid** [comment]
@uilianries one of the error
```
ConanException: VS non-existing installation: Visual Studio 16
```

seems unrelated to my change, could u trigger tests on develop branch, just to be sure they still work

**Comment by uilianries** [comment]
Appveyor is no longer used here, but I don't have full permission to remove that app from this repository.

I'll check Github actions.

**Comment by uilianries** [comment]
@boussaffawalid After a maintenance, the CI is working again. You is working good, merging it now. Thank you for your contribution.

**Pull request merged**